config/admin-cli.properties

```diff
@@ -19,11 +19,43 @@ cli.controller.grpc.uri=tcp://localhost:9090
 cli.store.metadata.backend=segmentstore
 
 # General configuration for the Pravega cluster at hand.
-pravegaservice.cluster.name=pravega/pravega
+pravegaservice.clusterName=pravega/bar-pravega
 pravegaservice.container.count=4
 
 # Config for Zookeeper service.
-pravegaservice.zk.connect.uri=localhost:2181
+pravegaservice.zk.connect.uri=zookeeper-client:2181
```

Comment:
Do these settings make sense or are they just for your local testing?

See line 29 below too.